lib/python/pyflyby/_autoimp.py

```diff
@@ -24,6 +25,8 @@
 from   pyflyby._modules         import ModuleHandle
 from   pyflyby._parse           import PythonBlock, infer_compile_mode
 
+if PY2:
+    long = int
```

Comment:
This seems less than ideal.  Why not just change the lone `long` call to `int`?

Reply:
I'm sure I meant to do this differently, but since it apparently works as an int rather than a long, I'll just remove it.